static/js/topic_zoom.js

```diff
@@ -36,6 +37,30 @@ export function zoom_out() {
     zoomed_in = false;
 }
 
+export function handle_topic_zoom_hotkey() {
+    if (is_zoomed_in()) {
+        if (stream_list.is_left_column_popover_hidden()) {
+            popovers.hide_all();
+            stream_popover.show_streamlist_sidebar();
+            document.querySelector("#filter-topic-input").focus();
+        } else {
+            zoom_out();
```

Comment:
I think it would be cleaner to have `zoom_out` call in `if` block and return early, meaning we won't ned else block.